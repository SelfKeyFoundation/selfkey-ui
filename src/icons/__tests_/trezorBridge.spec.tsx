import * as React from 'react';
import { create } from 'react-test-renderer';
import TrezorBridgeIcon from '../trezor-bridge';

describe('Trezor Bridge Icon', () => {
	it('should show Trezor Bridge Icon', () => {
		const component = create(<TrezorBridgeIcon />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
