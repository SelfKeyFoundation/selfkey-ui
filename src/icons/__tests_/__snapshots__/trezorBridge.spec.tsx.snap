// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trezor Bridge Icon should show Trezor Bridge Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-0-2-1"
  focusable="false"
  style={
    Object {
      "fill": "url(#default-linear-gradient)",
      "height": "66px",
      "width": "66px",
    }
  }
  viewBox="0 0 66 66"
>
  <defs>
    <linearGradient
      id="default-linear-gradient"
      x1="11.0842631%"
      x2="36.0705554%"
      y1="68.33943%"
      y2="0%"
    >
      <stop
        offset="0%"
        stopColor="#09A8BA"
      />
      <stop
        offset="100%"
        stopColor="#0ABBD0"
      />
    </linearGradient>
  </defs>
  <g
    fill="inherit"
    fillRule="evenodd"
    id="ðŸ› -Wallet-Setup"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="inherit"
      fillRule="nonzero"
      id="117.1---Trezor---Connecting-+-Messages"
      transform="translate(-368.000000, -1143.000000)"
    >
      <g
        id="Modal3"
        transform="translate(323.000000, 1029.000000)"
      >
        <path
          d="M74.9470699,114 L73.0285159,118.032618 C69.5341063,118.944851 66.4405389,120.7715 63.9814449,123.261914 L59.5212892,122.907422 L56.4683591,128.192578 L58.990625,131.855664 C58.5279452,133.534735 58.2,135.275312 58.2,137.1 C58.2,138.924688 58.5278946,140.665927 58.990625,142.344336 L56.4683591,146.007422 L59.5212892,151.292578 L63.9814449,150.938086 C66.4406797,153.428597 69.5349577,155.255122 73.0285159,156.167382 L74.9470699,160.2 L81.0529301,160.2 L82.9714841,156.167382 C86.4658937,155.255149 89.5594611,153.4285 92.0185551,150.938086 L96.4787108,151.292578 L99.5316409,146.007422 L97.009375,142.344336 C97.4720548,140.665265 97.8,138.924688 97.8,137.1 C97.8,135.275312 97.4721054,133.534073 97.009375,131.855664 L99.5316409,128.192578 L96.4787108,122.907422 L92.0185551,123.261914 C89.5593203,120.771403 86.4650423,118.944878 82.9714841,118.032618 L81.0529301,114 L74.9470699,114 Z M76.3371091,116.2 L79.6628909,116.2 L81.3923824,119.835157 L81.9123051,119.95332 C85.4171548,120.750113 88.5178634,122.592025 90.8841801,125.139648 L91.2451176,125.530664 L95.2691409,125.210547 L96.9320318,128.089453 L94.6611324,131.389453 L94.8179682,131.898632 C95.3252475,133.541594 95.6,135.28763 95.6,137.1 C95.6,138.91237 95.3253223,140.657126 94.8179682,142.301368 L94.6611324,142.810547 L96.9320318,146.110547 L95.2691409,148.989453 L91.2451176,148.671484 L90.8841801,149.060352 C88.5177545,151.608091 85.418224,153.44988 81.9123051,154.24668 L81.3923824,154.366993 L79.6628909,158 L76.3371091,158 L74.6076176,154.364843 L74.0876949,154.24668 C70.5828452,153.449887 67.4821366,151.607975 65.1158199,149.060352 L64.7548824,148.669336 L60.7308591,148.989453 L59.0679682,146.110547 L61.3388676,142.810547 L61.1820318,142.301368 C60.6747503,140.658407 60.4,138.91237 60.4,137.1 C60.4,135.28763 60.6746777,133.542874 61.1820318,131.898632 L61.3388676,131.389453 L59.0679682,128.089453 L60.7308591,125.210547 L64.7548824,125.528516 L65.1158199,125.139648 C67.4822455,122.591909 70.581776,120.75012 74.0876949,119.95332 L74.6076176,119.833007 L76.3371091,116.2 Z M76.9,127.2 L76.9,139.3 L72.5,139.3 L78,146.265234 L83.5,139.3 L79.1,139.3 L79.1,127.2 L76.9,127.2 Z M45,156.9 L45,174.5 C45,177.524468 47.4755324,180 50.5,180 L105.5,180 C108.524468,180 111,177.524468 111,174.5 L111,156.9 L108.8,156.9 L108.8,174.5 C108.8,176.335332 107.335332,177.8 105.5,177.8 L50.5,177.8 C48.6646676,177.8 47.2,176.335332 47.2,174.5 L47.2,156.9 L45,156.9 Z M53.8,156.9 C53.1924868,156.9 52.7,157.392487 52.7,158 C52.7,158.607513 53.1924868,159.1 53.8,159.1 C54.4075132,159.1 54.9,158.607513 54.9,158 C54.9,157.392487 54.4075132,156.9 53.8,156.9 Z M102.2,156.9 C101.592487,156.9 101.1,157.392487 101.1,158 C101.1,158.607513 101.592487,159.1 102.2,159.1 C102.807513,159.1 103.3,158.607513 103.3,158 C103.3,157.392487 102.807513,156.9 102.2,156.9 Z M53.8,161.3 C53.1924868,161.3 52.7,161.792487 52.7,162.4 C52.7,163.007513 53.1924868,163.5 53.8,163.5 C54.4075132,163.5 54.9,163.007513 54.9,162.4 C54.9,161.792487 54.4075132,161.3 53.8,161.3 Z M102.2,161.3 C101.592487,161.3 101.1,161.792487 101.1,162.4 C101.1,163.007513 101.592487,163.5 102.2,163.5 C102.807513,163.5 103.3,163.007513 103.3,162.4 C103.3,161.792487 102.807513,161.3 102.2,161.3 Z M53.8,165.7 C53.1924868,165.7 52.7,166.192487 52.7,166.8 C52.7,167.407513 53.1924868,167.9 53.8,167.9 C54.4075132,167.9 54.9,167.407513 54.9,166.8 C54.9,166.192487 54.4075132,165.7 53.8,165.7 Z M102.2,165.7 C101.592487,165.7 101.1,166.192487 101.1,166.8 C101.1,167.407513 101.592487,167.9 102.2,167.9 C102.807513,167.9 103.3,167.407513 103.3,166.8 C103.3,166.192487 102.807513,165.7 102.2,165.7 Z M53.8,170.1 C53.1924868,170.1 52.7,170.592487 52.7,171.2 C52.7,171.807513 53.1924868,172.3 53.8,172.3 C54.4075132,172.3 54.9,171.807513 54.9,171.2 C54.9,170.592487 54.4075132,170.1 53.8,170.1 Z M58.2,170.1 C57.5924868,170.1 57.1,170.592487 57.1,171.2 C57.1,171.807513 57.5924868,172.3 58.2,172.3 C58.8075132,172.3 59.3,171.807513 59.3,171.2 C59.3,170.592487 58.8075132,170.1 58.2,170.1 Z M62.6,170.1 C61.9924868,170.1 61.5,170.592487 61.5,171.2 C61.5,171.807513 61.9924868,172.3 62.6,172.3 C63.2075132,172.3 63.7,171.807513 63.7,171.2 C63.7,170.592487 63.2075132,170.1 62.6,170.1 Z M67,170.1 C66.3924868,170.1 65.9,170.592487 65.9,171.2 C65.9,171.807513 66.3924868,172.3 67,172.3 C67.6075132,172.3 68.1,171.807513 68.1,171.2 C68.1,170.592487 67.6075132,170.1 67,170.1 Z M71.4,170.1 C70.7924868,170.1 70.3,170.592487 70.3,171.2 C70.3,171.807513 70.7924868,172.3 71.4,172.3 C72.0075132,172.3 72.5,171.807513 72.5,171.2 C72.5,170.592487 72.0075132,170.1 71.4,170.1 Z M75.8,170.1 C75.1924868,170.1 74.7,170.592487 74.7,171.2 C74.7,171.807513 75.1924868,172.3 75.8,172.3 C76.4075132,172.3 76.9,171.807513 76.9,171.2 C76.9,170.592487 76.4075132,170.1 75.8,170.1 Z M80.2,170.1 C79.5924868,170.1 79.1,170.592487 79.1,171.2 C79.1,171.807513 79.5924868,172.3 80.2,172.3 C80.8075132,172.3 81.3,171.807513 81.3,171.2 C81.3,170.592487 80.8075132,170.1 80.2,170.1 Z M84.6,170.1 C83.9924868,170.1 83.5,170.592487 83.5,171.2 C83.5,171.807513 83.9924868,172.3 84.6,172.3 C85.2075132,172.3 85.7,171.807513 85.7,171.2 C85.7,170.592487 85.2075132,170.1 84.6,170.1 Z M89,170.1 C88.3924868,170.1 87.9,170.592487 87.9,171.2 C87.9,171.807513 88.3924868,172.3 89,172.3 C89.6075132,172.3 90.1,171.807513 90.1,171.2 C90.1,170.592487 89.6075132,170.1 89,170.1 Z M93.4,170.1 C92.7924868,170.1 92.3,170.592487 92.3,171.2 C92.3,171.807513 92.7924868,172.3 93.4,172.3 C94.0075132,172.3 94.5,171.807513 94.5,171.2 C94.5,170.592487 94.0075132,170.1 93.4,170.1 Z M97.8,170.1 C97.1924868,170.1 96.7,170.592487 96.7,171.2 C96.7,171.807513 97.1924868,172.3 97.8,172.3 C98.4075132,172.3 98.9,171.807513 98.9,171.2 C98.9,170.592487 98.4075132,170.1 97.8,170.1 Z M102.2,170.1 C101.592487,170.1 101.1,170.592487 101.1,171.2 C101.1,171.807513 101.592487,172.3 102.2,172.3 C102.807513,172.3 103.3,171.807513 103.3,171.2 C103.3,170.592487 102.807513,170.1 102.2,170.1 Z"
          id="trezor-bridge-icon"
        />
      </g>
    </g>
  </g>
</svg>
`;
